src/serviceWorker.ts

```diff
@@ -0,0 +1,10 @@
+export const service_worker = ()=>{
+    if ("serviceWorker" in navigator) {
+        navigator.serviceWorker.register("./sw.js").then(() => {
+          //Service Worker Registration Succesful
```

Comment:
Service worker registeration can be notified in console as it's more of a convention so please put
```js
console.log('Service Worker Registered');
```
Also handle .catch case for errors!